src/app.service.js

```diff
@@ -32,8 +32,14 @@ export class AppService {
     }
   }
 
-  getReadingTime(bodyElement, wpm) {
-    const mainContent = this.getMainContent(bodyElement);
+  getReadingTime(url, bodyElement, wpm) {
+    let mainContent = null;
+
+    if (url.includes("velog")) {
+      mainContent = this.getVelogMainContent(bodyElement);
+    } else {
+      mainContent = this.getMainContent(bodyElement);
+    }
 
     return this.calculateReadingTime(mainContent, wpm);
   }
```

Comment:
```suggestion
  getReadingTime(url, bodyElement, wpm) {
    if (url.includes("velog")) {
      const velogMainContent = this.getVelogMainContent(bodyElement);

      return this.calculateReadingTime(velogMainContent, wpm);
    }
    const mainContent = this.getMainContent(bodyElement);

    return this.calculateReadingTime(mainContent, wpm);
  }
```

```
// bad
function foo() {
  if (x) {
    return x;
  } else {
    return y;
  }
}

// good
function foo() {
  if (x) {
    return x;
  }

  return y;
}
```
불필요한 변수 선언은 피하고 코드의 가독성을 높이기 위해 no else return 규칙을 적용해 상기 코드와 같이 작성하는것은 어떨까요?
https://github.com/airbnb/javascript?tab=readme-ov-file#blocks

Reply:
해당 사항 반영하였습니다.